WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -379,6 +382,9 @@ public void onClick(View view) {
             }
         });
 
+        // determine how far the user has to scroll before the "new posts" bar is hidden
+        mHideNewPostsBarOnScrollOffset = DisplayUtils.dpToPx(context, 10);
+
```

Comment:
nitpicking here - wouldn't it be better to place the `10` constant on a static field?